meson.build

```diff
@@ -64,7 +64,7 @@ if not enable_trace
   add_project_arguments('-DVKD3D_NO_TRACE_MESSAGES', language : 'c')
 endif
 
-vkd3d_external_includes = [ './subprojects/Vulkan-Headers/include', './subprojects/SPIRV-Headers/include' ]
+vkd3d_external_includes = [ './subprojects/SPIRV-Headers/include' ]
```

Comment:
Looks like an unintentional change here.

If not, why were the header includes removed? We really don't want a dependency on whatever outdated crap the SDK is shipping.

Reply:
It was unintentional to include that. I will fix this and also update the commit message as requested in the other PR.